src/antsibull_docs/data/docsite/plugin.rst.j2

```diff
@@ -233,17 +235,19 @@ This describes the input of the test, the value before ``is @{plugin_name}@`` or
 {% endif %}
 
 {% if doc['positional'] and plugin_type in ['lookup', 'filter', 'test'] and doc['positional'] != ['_input'] %}
+{%   set has_positional_parameters = true %}
 {%   set options_to_skip = options_to_skip + doc['positional'] %}
 
 .. Positional
 
 Positional parameters
 ---------------------
 
+This describes positional parameters of the @{plugin_type}@. These are the values ``positional1``, ``positional2`` and so on in the following example:
 {%   if plugin_type == 'filter' %}
-This describes positional parameters of the filter. These are the values ``positional1``, ``positional2`` and so on in the following example: ``input | @{plugin_name}@(positional1, positional2, ...)``.
+``input | @{plugin_name}@(positional1, positional2, ...)``
 {%   elif plugin_type == 'test' %}
-This describes positional parameters of the test. These are the values ``positional1``, ``positional2`` and so on in the following examples: ``input is @{plugin_name}@(positional1, positional2, ...)`` and ``input is not @{plugin_name}@(positional1, positional2, ...)``.
```

Comment:
I could replace the "in the following example" with just a simple "e.g.:".
Or leave it as is by duplicating the whole line.

Reply:
`e.g.` is something we want to avoid in the Ansible docs (its a latin abbreviation). It might be easier to keep the duplicated line, or alternatively just move the word `example`/`examples` into the `{% if %}` branches and keep the remainder of the sentence outside.